VSCode/armsnippets.json

```diff
@@ -758,6 +758,213 @@
     ],
     "description": "Cosmos DB Database Account"
 },
+"Cosmos DB SQL Database": {
+    "prefix": "arm-cosmos-sql-database",
+    "body": [
+      "      {",
+      "            \"type\": \"Microsoft.DocumentDB/databaseAccounts/apis/databases\",",
+      "            \"name\": \"[concat(${1:account-name}, '/sql/', ${2:database-name})]\",",
```

Comment:
Don't these resource types require a location property?

Reply:
No, the location is enforced from the parent database account